src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -358,6 +358,16 @@ def set_olm_operator(self, operator_name, properties=None, update=False):
         self._config.olm_operators = olm_operators
         self.api_client.update_cluster(self.id, {"olm_operators": olm_operators})
 
+    def set_virt_bundle(self):
+        log.info(f"Setting virtualization bundle for cluster: {self.id}")
+        self.set_olm_operator("mtv")
+        self.set_olm_operator("nmstate")
+        self.set_olm_operator("node-healthcheck")
+        self.set_olm_operator("fence-agents-remediation")
+        self.set_olm_operator("kube-descheduler")
+        self.set_olm_operator("self-node-remediation")
+        self.set_olm_operator("node-maintenance")
+
```

Comment:
@eliajahshan  , yes we can use it and it will be managed here when something change...  i think you can use const.OperatorType as naming.

@danmanor  Is it ok ? we are going to switch to bundles

Reply:
Yes, have to tested it @eliajahshan ?